extension/src/test/fixtures/expShow/base/customPlots.ts

```diff
@@ -9,7 +9,7 @@ import { Experiment } from '../../../../experiments/webview/contract'
 export const customPlotsOrderFixture: CustomPlotsOrderValue[] = [
   {
     metric: 'summary.json:loss',
-    param: 'params.yaml:dropout'
+    param: 'params.yaml:log_file'
```

Comment:
I updated the fixture to include a string param for testing but this did lead me to touching practically every custom plots test, despite this being a rather small spec change 😅